.github/workflows/unit-tests-pr.yml

```diff
@@ -23,3 +25,5 @@ jobs:
     name: Unit Tests
     if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
     uses: ./.github/workflows/frontend-unit-tests.yml
+    secrets:
+      codecov_token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This token is provided in https://app.codecov.io/gh/yeagerai/genlayer-simulator/settings. I've added it to GH Secrets in this repo